### ISSUE TITLE
Support for 0.17 

### ISSUE THREAD

**Issue by npo6ka** [Factorio-Xander-Mod/Xander-Mod#70]
It would be nice to have support for 0.17.

**Comment by jodokus31** [comment]
I did some changes in https://github.com/Factorio-Xander-Mod/Xander-Mod/tree/update_0_17 , which at least is able to start the game with version 0.17.6 and load a new or existing map. There were a fair bunch of changes and I solved it quick and dirty.
The graphic packs must also be changed by updating the factorio version in info.json

Maybe Repofme1 wants to merge it or use it or a least as indicator, where things need to be fixed.

**Comment by npo6ka** [comment]
I also updated the new version, however I took the repository with a large number of corrections:
https://github.com/npo6ka/Xander-Mod/

**Comment by npo6ka** [comment]
Maybe you should consider adding some rights from the Mathematics7 repository to the main repository Factorio-Xander-Mod?

**Comment by jodokus31** [comment]
Yes, good idea. Also your solutions for 0.17 seem a bit cleaner and more complete. Unfortunately, this is based on Xander version 1 and there were some big changes in the meantime, new resources, list based item and recipes, etc., so all fixes have to be analysed individually...

**Comment by ipcyborg** [comment]
It looks kind of strange to use KJ here:

Xander-Mod/prototypes/entity/logistic/inserter.lua
{
	name = "inserter-slow",
	...
	energy_per_movement = "3000J",
	energy_per_rotation = "3000KJ",	   <- should be "3000J"?
	...
}

**Comment by jodokus31** [comment]
Oops, sure. Thanks

**Comment by Ranakastrasz** [comment]
> I also updated the new version, however I took the repository with a large number of corrections:
> https://github.com/npo6ka/Xander-Mod/

Is there a reason I can't post issues on that fork?
And which one is the more 'semi-official' one anyway? There is cooperation, right?

**Comment by jodokus31** [comment]
This fork is based on Xander Version 1, which is actually updated to 0.17 on mod portal:
https://github.com/Mathematics7/Xander-Mod

This one here is "official", because its hosted by RepofMe1, the originator of the mod, but it is currently not finally ported to 0.17. It also contains some unfinished major changes compared to Xander Version 1. He is currently on a hiatus
I was helping in the past a bit and put a small bit of effort to port it to 0.17 in a new branch, but i have not much time to test and polish it.
